clientGuide.md

```diff
@@ -0,0 +1,100 @@
+HTTP Wrapper
+============
+
+The [Bloomberg Open API] provides access to market data. The
+[Bloomberg Open API HTTP Wrapper] makes the API available via HTTP: clients
+can POST commands to an HTTP server running the wrapper, which will respond 
+with the relevant data.
+
+The HTTP wrapper is currently in development, and does not yet provide access
+to all features of the API. Notably, it currently supports the
+Request/Response paradigm of the API, but not Subscription.
+
+This document describes the HTTP wrapper itself: the URLs to use and the
+formatting for the HTTP requests and responses. It also includes basic info
+about some of the underlying API operations. For more details, refer to the
+[BLPAPI Developer's Guide]. This document will often refer to specific
+sections of the Developer's Guide where appropriate.
+
+[Bloomberg Open API]: http://bloomberglabs.com/api
+[Bloomberg Open API HTTP Wrapper]: http://github.com/bloomberg/blpapi-http
+[BLPAPI Developer's Guide]: http://www.bloomberglabs.com/files/2014/07/blpapi-developers-guide-2.54.pdf
+
+
+Requesting data
+---------------
+
+To request data, a client makes an HTTP POST request on a URL of the following
+form:
+
+http://blpapihost.example.com/v1.0/request/blp/refdata/{operation}
```

Comment:
As discussed, leaving as-is for now, but will definitely expand to more general forms when we support more stuff in future versions.